addons/gamification/views/res_users_views.xml

```diff
@@ -7,9 +7,12 @@
         <field name="model">res.users</field>
         <field name="inherit_id" ref="base.view_users_form"/>
         <field name="arch" type="xml">
-            <group name="messaging" position="inside">
-                <field name="karma"/>
-            </group>
+            <xpath expr="//div[@name='button_box']" position="inside">
+                <button type="object" class="oe_stat_button" groups="base.group_no_one"
```

Comment:
Why debug mode? Could be nice to always have the karma displayed, but maybe there are a lot of buttons already.